scrapers/Babes.yml

```diff
@@ -0,0 +1,224 @@
+name: Babes
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.babes.com/scene
+    scraper: sceneScraper
+#sceneByName:
+#  action: scrapeXPath
+#  queryURL: https://www.babes.com/scenes?q={}
+#  scraper: sceneScraper
+#sceneByQueryFragment:
+#  action: scraperXPath
+#  queryURL: "{url}"
+#  scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - www.babes.com/model
+    scraper: performerScraper
+#performerByName:
+#  action: scrapeXPath
+#  queryURL: https://www.babes.com/models?q={}
+#  scraper: performerSearch
+xPathScrapers:
+#  performerSearch:
+#    performer:
+#      Name: //a[contains(concat(' ',normalize-space(@class),' '),' one-list-hueuj4 ')]/text()
+#      URL:
+#        selector: //a[contains(concat(' ',normalize-space(@class),' '),' one-list-hueuj4 ')]/@href
+#        postProcess:
+#          - replace:
+#              - regex: ^
+#                with: https://www.babes.com
+  performerScraper:
+    common:
+      $root: //div[contains(concat(' ',normalize-space(@class),' '),' cPKtEE ')]
```

Comment:
These CSS classes are regenerated every time the website deploys a new version so we cannot rely on them staying stable for any amount of time

Reply:
Thanks for the info Maista. I did read your other reply and will totally stand by while you guys decide what to do. Out of interest, do you know of a better or alternate method for extracting the desired material not relying on the CSS class?

Reply:
It's tricky and arguably less readable, but we can rely on the structure of the page instead: this means using the kinds of XPath selectors you get from right-clicking an element in Developer Tools and choosing `Copy > XPath`. Here's an example of selectors for the "scene element" at babes.com:
```
/html/body/div/div[1]/div[2]/div[2]/div[2]/div[1]
```

So if we use that as a common fragment we can select the description using this selector
```
$scene/div[2]/section/div/p
```